internal/build/util.go

```diff
@@ -197,7 +197,7 @@ func ExpandPackagesNoVendor(patterns []string) []string {
 		}
 		var packages []string
 		for _, line := range strings.Split(string(out), "\n") {
-			if !strings.Contains(line, "/vendor/") {
+			if !strings.Contains(line, "/vendor/") && !strings.Contains(line, "go: downloading") {
```

Comment:
why do we need this fix?

Reply:
because this step is downloading the go-fuzz library and there is an output.
Therefore the first package is "go downloading: github.com/google/go-fuzz" and it is not a package, and it fails in the next step (go test)

Reply:
but why it doesn't fail for go-ethereum? as they don't have this change and they also use go-fuzz.

Reply:
if you already have the package downloaded in your go/pkg, there is no issue, as it is already present.
A commit before I used a prior `go mod download` without modifying ci.go.
https://github.com/ConsenSys/quorum/runs/2896855122?check_suite_focus=true#step:5:723

I am not sure go-ethereum use this github action environment.

Reply:
My intuition is that go build doesn't build the tests, and this package is not downloaded at this stage.

Reply:
@baptiste-b-pegasys add a `Quorum` tag on it (as we modified) and the reason for the modification